### PR TITLE
Filebash

### PR DESCRIPTION
**Pull request by Klavdia27**

### CODE REVIEW

File: scripts/ask_adult.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# код языка
+#!/bin/bash 
+# вывод в командную строку сообщения
+echo -n "Введите значение переменной a  "
```
Comment (vladislavkovaliov) [inline scripts/ask_adult.sh:4]: `-n` need to be removed

Comment (vladislavkovaliov) [inline scripts/ask_adult.sh]: No need `-n` in a terminal
image

File: scripts/ask_adult.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# код языка
+#!/bin/bash 
+# вывод в командную строку сообщения
+echo -n "Введите значение переменной a  " 
+# запись ввода из командной строки в переменную а
+read a
```
Comment (vladislavkovaliov) [inline scripts/ask_adult.sh]: I guess that variable should have a proper name. You can use `age`, for example.

File: scripts/ask_adult.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#оператор if  проверяет число ил нет
+if [ -n "$a" ] && [ "$a" -eq "$a" ] 2>/dev/null; then
+# оператор if и его условие
+if [ "$a" -ge 18 ]
```
Comment (vladislavkovaliov) [inline scripts/ask_adult.sh]: You have to use code styling `https://ru.wikipedia.org/wiki/%D0%A1%D1%82%D0%B0%D0%BD%D0%B4%D0%B0%D1%80%D1%82_%D0%BE%D1%84%D0%BE%D1%80%D0%BC%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F_%D0%BA%D0%BE%D0%B4%D0%B0`

File: scripts/hello_world.bat [hunk 4]
```diff
@@ -0,0 +1,3 @@
+@echo off           # чтоб команды не отображались на экране
```
Comment (vladislavkovaliov) [inline scripts/hello_world.bat]: This file should be deleted.

File: scripts/helloworldvariable.bat [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+@echo off
```
Comment (vladislavkovaliov) [inline scripts/helloworldvariable.bat]: This file should be deleted.

File: scripts/ask_adult.sh (hunk 1 above)
Comment (Klavdia27) [reply scripts/ask_adult.sh]: made changes

File: scripts/ask_adult.sh (hunk 2 above)
Comment (Klavdia27) [reply scripts/ask_adult.sh]: made changes

File: scripts/hello_world.bat (hunk 4 above)
Comment (Klavdia27) [reply scripts/hello_world.bat]: made changes

File: scripts/helloworldvariable.bat (hunk 5 above)
Comment (Klavdia27) [reply scripts/helloworldvariable.bat]: made changes

File: scripts/hello_world.bat (hunk 4 above)
Comment (Klavdia27) [reply scripts/hello_world.bat]: hello

File: scripts/ask_adult.sh (hunk 3 above)
Comment (Klavdia27) [reply scripts/ask_adult.sh]: hello

Comment (Klavdia27) [reply scripts/ask_adult.sh]: made changes

**Pull request merged**